st2common/st2common/util/crypto.py

```diff
@@ -263,15 +267,18 @@ def cryptography_symmetric_decrypt(decrypt_key, ciphertext):
 
     NOTE 2: This function is loosely based on keyczar AESKey.Decrypt() (Apache 2.0 license).
     """
-    assert isinstance(decrypt_key, AESKey), 'decrypt_key needs to be AESKey class instance'
-    assert isinstance(ciphertext, (six.text_type, six.string_types, six.binary_type)), \
-        'ciphertext needs to either be a string/unicode or bytes'
+    if not isinstance(decrypt_key, AESKey):
+        raise ValueError('Decrypted key needs to be an AESKey class instance.')
+    if not isinstance(ciphertext, (six.text_type, six.string_types, six.binary_type)):
+        raise TypeError('Ciphertext needs to either be a string/unicode or bytes.')
 
     aes_key_bytes = decrypt_key.aes_key_bytes
     hmac_key_bytes = decrypt_key.hmac_key_bytes
 
-    assert isinstance(aes_key_bytes, six.binary_type)
-    assert isinstance(hmac_key_bytes, six.binary_type)
+    if not isinstance(aes_key_bytes, six.binary_type):
+        raise TypeError('The aes key bytes does not match with binary type.')
+    if not isinstance(hmac_key_bytes, six.binary_type):
+        raise TypeError('The hmac key bytes does not match with binary type.')
```

Comment:
Same issues here - binary type can just be bytes, and include what type was given.